mods/ra/maps/soviet-07/soviet07.lua

```diff
@@ -0,0 +1,346 @@
+		Actor.Create("ftur", true, { Owner = player, Location = FTur2Goal.Location})
+		Camera.Position = CameraGoalCenter1.CenterPosition
+		if not cameraGoalRightTrigger then
+			Actor.Create("camera", true, { Owner = player, Location = CameraGoalCenter1.Location })
+			Actor.Create("camera", true, { Owner = player, Location = CameraGoalCenter2.Location })
+			Actor.Create("camera", true, { Owner = player, Location = CameraGoalCenter3.Location })
+		end
+		Utils.Do(GoalGuards, function(actor)
+			actor.Attack(FTur1Goal)
+		end)
+		player.MarkCompletedObjective(sovietObjective4)
+	end
+end)
+
+fTurBottomTrigger = false
+Trigger.OnEnteredFootprint(FTurBottomTrigger, function(a, id)
+	if not fTurBottomTrigger and a.Owner == player then
+		fTurBottomTrigger = true
+		if not rSoldierTrapTrigger then
+			Actor.Create("camera", true, { Owner = player, Location = CameraRSoldier.Location })
+			Actor.Create("camera", true, { Owner = player, Location = CameraFTurBottom.Location })
+		end
+	end
+end)
+
+fTurLeftTrigger = false
+Trigger.OnEnteredFootprint(FTurLeftTrigger, function(a, id)
+	if not fTurLeftTrigger and a.Owner == player then
+		fTurLeftTrigger = true
+		Actor.Create("camera", true, { Owner = player, Location = CameraFTurLeft.Location })
+	end
+end)
+
+fTurRightTrigger = false
+Trigger.OnEnteredFootprint(FTurRightTrigger, function(a, id)
+	if not fTurRightTrigger and a.Owner == player then
+		fTurRightTrigger = true
+		Actor.Create("camera", true, { Owner = player, Location = CameraFTurRight.Location })
+	end
+end)
+
+goalCenterTrigger = false
+Trigger.OnEnteredFootprint(GoalCenterTrigger, function(a, id)
+	if not goalCenterTrigger and a.Owner == player and a.Type == "e6" then
+		goalCenterTrigger = true
+		player.MarkCompletedObjective(sovietObjective5)
+	end
+end)
+
+goalLeft1Trigger = false
+Trigger.OnEnteredFootprint(GoalLeft1Trigger, function(a, id)
+	if not goalLeft1Trigger and a.Owner == player and a.Type == "e6" then
+		goalLeft1Trigger = true
+		Media.PlaySpeechNotification(player, "ControlCenterDeactivated")
+	end
+end)
+
+goalLeft2Trigger = false
+Trigger.OnEnteredFootprint(GoalLeft2Trigger, function(a, id)
+	if not goalLeft2Trigger and a.Owner == player and a.Type == "e6" then
+		goalLeft2Trigger = true
+		Media.PlaySpeechNotification(player, "ControlCenterDeactivated")
+	end
+end)
+
+goalRight1Trigger = false
+Trigger.OnEnteredFootprint(GoalRight1Trigger, function(a, id)
+	if not goalRight1Trigger and a.Owner == player and a.Type == "e6" then
+		goalRight1Trigger = true
+		Media.PlaySpeechNotification(player, "ControlCenterDeactivated")
+	end
+end)
+
+goalRight2Trigger = false
+Trigger.OnEnteredFootprint(GoalRight2Trigger, function(a, id)
+	if not goalRight2Trigger and a.Owner == player and a.Type == "e6" then
+		goalRight2Trigger = true
+		Media.PlaySpeechNotification(player, "ControlCenterDeactivated")
+	end
+end)
+
+rSoldierTrap1killed = false
+rSoldierTrap2killed = false
+rSoldierTrapTrigger = false
```

Comment:
You can remove the three lines above.
The behaviour will be the same, as `if not foo` will also be true when you don't explicitly initialize `foo` as `false`.

Same for all other `= false`'s.